src/main/java/fitr/parser/Parser.java

```diff
@@ -52,4 +55,11 @@ public static Command parse(String userInput) {
             return new InvalidCommand(arguments);
         }
     }
+
+    public static String getCurrentDate() {
+        Calendar calendar = Calendar.getInstance();
+        Date currentDateTime = calendar.getTime();
+        SimpleDateFormat dateFormatter = new SimpleDateFormat("d/M/y");
+        return dateFormatter.format(currentDateTime);
+    }
```

Comment:
Would it be possible to put this in a new class? Something like a `DateTimeManager` class?